llpc/test/shaderdb/fuzzer/GraphicsFuzz_TestOpCopyObjectFromAccessChain2.spvasm

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Is it possible to rename the tests and remove the postfix "2". I don't want we have a bunch of tests that are XXX, XXX1, XXX2, XXX3. Meaningful names are more helpful.

Reply:
I see your point.  However, what is meaningful about these tests is that they exposed crashes in LLPC - the represent the minimized SPIR-V that could reproduce the associated crash.  Beyond that, the specific SPIR-V that they contain isn't particularly meaningful.

I can see two options:

(1) Give them a name inspired by the instructions that remain after minimisation, e.g. "ExtractAndCopyObject" if the test involves a combination of OpCompositeExtract and OpCopyObject.  

(2) Give them a name containing details of the problem that they triggered, e.g. "InlineAttributeCompatibilityProblem" for #923.

What do you think would be best, @amdrexu ?

Reply:
Thank you for explanations. I think option 2 is more reasonable, such as XXX_TestInlineAttributeCompatibility. I think the postfix "problem" is unnecessary. All fuzz tests are to catch problems :)

Reply:
I have renamed the tests.